react-js/src/subviews/loading-screen.tsx

```diff
@@ -0,0 +1,20 @@
+import {CircularProgress} from "@material-ui/core";
+import React from "react";
+
+export default class LoadingScreen extends React.Component<any, any> {
```

Comment:
loading screen should be shown for every long-lasting operation, e.g. filtering, cropping. is it?